typings/config.d.ts

```diff
@@ -107,6 +117,13 @@ declare global {
         auditRefs: AuditRef[];
       }
       export interface Group extends GroupJson {}
+
+      export type MergeOptionsOfItems = <T extends {path?: string, options: Record<string, any>}>(items: T[]) => T[];
```

Comment:
good news is with the latest tsc changes, these will be able to live in jsdocs soon